src/App.js

```diff
@@ -7,30 +7,34 @@ const { default: CreateMuiTheme } = require('@material-ui/core/styles/createMuiT
 const { default: MuiThemeProvider } = require('@material-ui/styles/ThemeProvider');
 const { default: CssBaseline } = require('@material-ui/core/CssBaseline');
 const StrangeRouter = require('strange-router');
+const MiddleEnd = require('strange-middle-end');
```

Comment:
Do we like `MiddleEnd`, or should we call this something else, like `ReactMiddleEnd`?

Reply:
Looks fine to me!